src/style.css

```diff
@@ -1 +1,79 @@
-/* Styles go here */
+body {
+  background: #e5e5e5;
+  margin: 0;
+  min-width: 375px;
+  max-width: 1200px;
+}
+
+.header {
+  box-sizing: border-box;
+  justify-content: space-between;
+  background-color: #fff;
+  display: flex;
+  align-items: center;
+  font-family: "Roboto", sans-serif;
+  padding-left: 50px;
+  padding-right: 50px;
+}
+
+.logo {
+  font-size: 0;
+  padding-top: 6px;
+  padding-bottom: 6px;
+}
+
+.nav {
+  position: relative;
+  margin: 0;
+}
+
+.nav__list {
+  display: flex;
+  cursor: pointer;
+  text-transform: uppercase;
+  margin: 0;
+}
+
+.nav__item {
+  list-style: none;
+  text-decoration: none;
+  text-align: center;
+  margin-left: 20px;
+}
+
+.nav__link {
+  color: #000;
+  text-decoration: none;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 60px;
+}
+
+.nav__link::after {
+  display: block;
+  content: "";
+  background: #00acdc;
+  border-radius: 8px;
+  transition: 300ms;
+  height: 4px;
+  position: absolute;
+  bottom: 0;
+}
+
+.nav__link:hover::after {
```

Comment:
It's wrong, you not need this
![image](https://user-images.githubusercontent.com/54936044/228374816-e56822be-09a2-4aab-9b1c-29263adfa5c9.png)